tasks/handle_container.yml

```diff
@@ -0,0 +1,120 @@
+- name: Update container
+  containers.podman.podman_container: "{{ __container }}"
+  vars:
+    __container: "{{ __podman_container | dict2items |
+      rejectattr('key', 'match', 'generate_systemd') |
+      items2dict }}"
+  register: __podman_container_info
+
+- name: Ensure systemd unit file
+  when: __podman_container.get('generate_systemd')
+  vars:
+    __systemd_params: "{{ __podman_container['generate_systemd']
+      if __podman_container['generate_systemd'] is mapping
+      else dict() }}"
+    __basename: "{{ __systemd_params.get('container_prefix', 'container') ~
+      __systemd_params.get('separator', '-') ~
+      __podman_container['name'] ~ '.service' }}"
+    __systemd_unit_filename: "{{
+      __systemd_params.get('path', '/etc/systemd/system')
```

Comment:
Will we add support for non-root users later on?

Reply:
Yes.